content/blog/2019/cve-2019-12995/index.md

```diff
@@ -0,0 +1,96 @@
+A bug in Istio’s JWT validation filter causes Istio Proxy to crash in certain cases when the request contains a malformed JWT token. The bug was discovered and reported by a user [on GitHub](https://github.com/istio/istio/issues/15084) on June 23, 2019.
+
+This bug affects all versions of Istio that are using the JWT authentication policy.
+
+The symptoms of the bug is an HTTP 503 error seen by the client, and
```

Comment:
sympton, singular

Reply:
no, it's symptoms they are two (and thus, the real comment is : symptoms of the bug *are*).
503 from client + log line in Envoy.